index.js

```diff
@@ -20,8 +20,8 @@ module.exports = {
   "rules": {
     "no-unexpected-multiline": 1,
     "semi": [1, "never"],
-
     "comma-dangle": [ 1, "always-multiline" ],
+    "no-case-declarations": 1,
```

Comment:
helps avoid the const/let hoisting issue of `case` statements. See the "OR" part of https://github.com/kylpo/react-playbook/blob/master/best-practices/redux.md#reducers for more